lynx/plan.sh

```diff
@@ -0,0 +1,23 @@
+pkg_origin=core
+pkg_name=lynx
+pkg_version=2.8.8
+pkg_description="Lynx is the text web browser."
+pkg_upstream_url=http://lynx.browser.org/
+pkg_license=("GPL-2.0")
+pkg_maintainer="The Habitat Maintainers <humans@habitat.sh>"
+pkg_source="http://invisible-mirror.net/archives/$pkg_name/tarballs/$pkg_name${pkg_version}rel.2.tar.gz"
```

Comment:
agreed, most of the original plans in the repo use `${variable}`

Some SO comments on the subject: http://unix.stackexchange.com/questions/4899/var-vs-var-and-to-quote-or-not-to-quote